.spelling

```diff
@@ -58,20 +67,27 @@ iOS
 JSON
 JSONSchema
 Jupyter
+lookups
 Lua
 matcher
 matchers
 metadata
 metastore
+Mocli
 MozTelemetry
 MySQL
+namespace
+namespaces
+natively
 nbconvert
 nbviewer
 nginx
 ni
 non-e10s
+NoSQL
 npm
 NumPy
+optionally-gzipped
```

Comment:
I don't think we need this exception, the sentence in question reads well without the hyphen: 

```
Data arrives as an HTTP POST of an optionally gzipped payload of JSON.
```